src/App.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div class="container">
+    <div class="row">
+      <div class="col-4">
+        <MemoList @memoSelected="memoSelected" :memos="memos"></MemoList>
+        <AddButton @openNewMemo="openNewMemo"></AddButton>
+      </div>
+      <div class="col-8">
+        <EditForm v-if="editing || adding" @saveMemo="saveMemo" @delMemo="delMemo" :memo="this.editingMemo"></EditForm>
```

Comment:
ここは this を書かなくても動くと思うので this は削っちゃても良いかもです ✂️ 

```suggestion
        <EditForm v-if="editing || adding" @saveMemo="saveMemo" @delMemo="delMemo" :memo="editingMemo"></EditForm>
```